src/ctia/schemas/search_agg.clj

```diff
@@ -30,7 +30,14 @@
   (st/optional-keys
    {:filter-map   {s/Keyword s/Any}
     :range        RangeQuery
-    :full-text    FullTextQuery}))
+    :full-text    (s/conditional map? FullTextQuery :else [FullTextQuery])}))
```

Comment:
The only change in this ns should be
```suggestion
    :full-text    [FullTextQuery]}
```
Since this code is only used in CTIA, you then need to update every dependencies to this schema.